packages/angular/src/app/shared/components/contact-status/contact-status.component.ts

```diff
@@ -7,13 +7,15 @@ import { ContactStatus } from 'src/app/shared/types/contact';
 @Component({
   selector: 'contact-status',
   template: `
-  <span class="{{ input && 'input' }} status status-{{ value | lowercase }}">{{ value }}</span>
+  <span class="{{ input && 'input' }} status status-{{ value | lowercase }}">{{ showText ? value : '' }}</span>
```

Comment:
it's seems that class input is not used

Reply:
I agree with you. I will remove this class and input prop